content/SCALE/SCALETutorials/Shares/NVMe-oF/_index.md

```diff
@@ -0,0 +1,174 @@
+* **NVMe-oF** service option on the **System > Services** screen
+
+The **NVMe-oF Global Configuration** shows the base NQN for the service.
+
+{{< expand "What is the base NQN?" "v" >}}
+The *base NQN* us the standardized NVMe Qualified Name for the service.
+The NQN standard structure follows the format defined in the base specification. The maximum length/size is 223 bytes.
+TrueNAS subsystems use the base NQN as the root identifier for the NVMe subsystems in fabric deployments.
+Discovery controllers use the standardized NQN to advertise available subsystems.
+Storage systems use the base NQN to authenticate and authorize host connections.
+The base NQN format provides the foundation for all NVMe-oF naming, ensuring interoperability, and preventing naming conflicts across different vendors and implementations.
+{{< /expand >}}
+
+{{< trueimage src="/images/SCALE/Shares/NVMeoFGlobalConfigurationScreen.png" alt="NVMe-oF Global Configuration Screen" id="NVMe-oF Global Configuration Scree" >}}
+
+TrueNAS populates the **baseNQN** field with the NVMe identifier.
+Accept this value or click in the field to copy/paste a new, properly formatted base NQN identifier.
+NQN is used as the prefix when creating a subsystem, if a subnqn is not supplied.
+Modifying this value does not change the subnqn of any existing subsystems.
+
+Select **Generate Cross-port Referrals for Ports on This System**  to allow xport_referral.
+If ANA is active, referrals are always generated between the peer ports on each TrueNAS controller node.
+
+Click **Save** to save changes and close the screen.
+
+### Adding Remote Direct Memory Access (RDMA)
+
+Go to the [**NVMe-oF Global Configuration**](#configuring-the-nvme-of-service) screen.
+
+Select the **Enable Remote Direct Memory Access (RDMA)** option. Click **Save**.
+
+Enabling RDMA allows configuring one or more ports with RDMA selected as the transport when enabled.
+This option requires an Enterprise license, and you must have an RDMA-capable system and network equipment.
+
+This option is inactive on systems without an Enterprise licenses.
+If the system is not equipped with required hardware, shows **Not enabled, because this system does not support RDMA** on the screen.
+
+### Adding Asymmetric Namespace Access (ANA)
+
+Go to the [**NVMe-oF Global Configuration**](#configuring-the-nvme-of-service) screen.
+Select the **Enable Asymmetric Namespace Access (ANA)** option, and click **Save**.
+
+This allows storage systems to inform hosts about the optimal controller path to access a namespace on Enterprise licensed systems.
+It is equivalent to Asymmetric Logical Unit Access (ALUA) in iSCSI.
+
+ANA helps storage arrays communicate to hosts which controller provides the best (lowest latency) path to specific namespaces, enabling intelligent multipathing and improved performance in NVMe-oF environments.
+
+## Adding a Subsystem
+
+Subsystems correlate to iSCSI targets. Each subsystem has a namespace, host, and port.
```

Comment:
Subsystems correlate to iSCSI targets. Each subsystem has at least one namespace and port (and host).

Dunno how to phrase that well.  We *need* a namespace and a port in order to be useful.  hosts are optional (for security).